### ISSUE TITLE
Path to 1.0.1

### ISSUE THREAD

**Issue by rkosafo** [rspeele/Rezoom.SQL#42]
This is to track issues/tasks on the path to 1.0.1.

- [ ] Upgrade *Rezoom* to 1.0.1 (package not available)
- [ ] Upgrade *FParsec-Pipes* to 1.1.1 (gives an error)
- [x] Test TSQL
- [x] Test Sqlite
- [x] Test Posgres
- [x] Fix error when select has only 1 field

**Comment by rkosafo** [comment]
@rspeele I get the error below when it was updated to *FParsec-Pipes 1.1.1*. *FParsec-Pipes 1.1.1* works fine.

 ````FileLoadException: Could not load file or assembly 'FParsecCS, Version=1.0.3.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. A strongly-named assembly is required. (Exception from HRESULT: 0x80131044)````

I'm suspecting it has something to do with *A strongly-named assembly is required*. Any idea if there is any difference between the libraries besides the version numbers. Could it be signing related?

**Comment by rspeele** [comment]
Haven't had much chance to look at this one, but I made a console app in .NET Framework and .NET Core referencing FParsec-Pipes and wasn't able to get the same problem to occur. Nothing changed between 1.1.0 and 1.1.1 other than this commit: https://github.com/rspeele/FParsec-Pipes/commit/69b62b5f3f1e5e1a611f8363b397f304a8693401

**Comment by rkosafo** [comment]
Is the library signed? Could that be a reason or vice versa?

**Comment by rspeele** [comment]
Yes, fparsec-pipes is signed. The FParsecCS assembly is part of the base
FParsec library but I don't know why it wouldn't be signed - it looks like
it is when I reference it from a console app.

**Comment by rspeele** [comment]
Closed with recent revamp, new package versions across the board, working in net10.0. @rkosafo you may or may not still be on github but I always appreciated your work and interest in this project!
